README.md

```diff
@@ -2,7 +2,7 @@
 Multi-Domain Wizard-of-Oz dataset (MultiWOZ), a fully-labeled collection of human-human written conversations spanning over multiple domains and topics. At a size of 10k dialogues, it is at least one order of magnitude larger than all previous annotated task-oriented corpora.
 
 
-The newest, corrected version of the dataset is available at [MultiWOZ_2.2](https://github.com/budzianowski/multiwoz/blob/master/data/MultiWOZ_2.2) thanks to [the Google crew](https://arxiv.org/abs/2007.12720).
+The newest, corrected version of the dataset is available at [MultiWOZ_2.2](https://github.com/budzianowski/multiwoz/blob/master/data/MultiWOZ_2.2.zip) thanks to [the Google crew](https://arxiv.org/abs/2007.12720).
```

Comment:
Please revert this.

Reply:
I have reverted this change in the new commit "Revert unnecessary changes in readme." If you have further questions, please let me know. Thank you.